### ISSUE TITLE
Terrain 'pokes through' large corridors

### ISSUE THREAD

**Issue by denverpierce** [AnalyticalGraphicsInc/cesium#4285]
Seen on the [CZML corridor sandcastle](http://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=CZML%20Corridor.html&label=CZML) with terrain on.

Terrain clips through the red corridor for tall mountains in the Rockies.

**Comment by hpinkos** [comment]
Thanks @denverpierce, I can reproduce this

![image](https://cloud.githubusercontent.com/assets/3451886/18290908/5b7d872c-7453-11e6-951f-fff53074c64c.png)

@bagnell do you think this is related to #4161?

**Comment by bagnell** [comment]
No. This is more likely a problem with getting the min and max heights of the terrain a long the corridor.

**Comment by pjcozzi** [comment]
Related to #4751, but that only fixes the bottom cap.

@bagnell perhaps this is because terrain is loaded after the corridor is created?  Or the wrong terrain LOD is used?
